generate.py

```diff
@@ -0,0 +1,50 @@
+from random import choice
```

Comment:
generate is action, while a module is an object holding some features. So `generator` is better.

Reply:
Understood. Thanks.